src/Plugins/BotSharp.Plugin.LLamaSharp/BotSharp.Plugin.LLamaSharp.csproj

```diff
@@ -11,7 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LLamaSharp" Version="0.12.0" />
+    <PackageReference Include="LLamaSharp" Version="0.13.0" />
+    <PackageReference Include="LLamaSharp.Backend.Cpu" Version="0.13.0" />
```

Comment:
Please remove below native libraries, it should be installed in the startup application, not in this library.
`LLamaSharp.Backend.Cpu`
`LLamaSharp.Backend.Cuda11`
`LLamaSharp.Backend.Cuda12`
`LLamaSharp.Backend.MacMetal`
`LLamaSharp.Backend.OpenCL`